README.md

```diff
@@ -7,47 +7,107 @@
 
 Drip campaigns are pre-written sets of emails sent to customers or prospects over time. Django Drips lets you use the admin to manage drip campaign emails using querysets on Django's User model.
 
-This project is a fork of the one written by [Zapier](https://zapier.com/z/qO/) and it's intended to run with Django 3.0.
+This project is a fork of the one written by [Zapier](https://zapier.com/z/qO/).
 
-[Read the docs](https://django-drip.readthedocs.org/en/latest/) or [check out a demo](http://djangodrip.com/).
+## Installation:
 
-### Installing:
-
-We highly recommend using pip to install _django-drip_, the packages are regularly updated
-with stable releases:
+1. Install django-drip-campaings using pip:
 
 ```
 pip install django-drip-campaigns
 ```
 
-Next, you'll want to add `drip` to your `INSTALLED_APPS` in settings.py.
+2. Add `'drip'` to your `INSTALLED_APPS` list on your settings.
 
 ```python
-INSTALLED_APPS = (
+INSTALLED_APPS = [
     'django.contrib.contenttypes',
     'django.contrib.comments',
     'django.contrib.sessions',
     'django.contrib.sites',
     'django.contrib.admin',
 
-    # Your favorite apps
+    # ...
 
     'drip',
-)
+]
 ```
 
-Don't forget to add `DRIP_FROM_EMAIL` to settings.py, or else we will fall back to `EMAIL_HOST_USER`.
+3. (Optional) Set `DRIP_FROM_EMAIL = '<your_app_from_email>'` in your settings, where `<your_app_from_email>` is the email account that is going to be shown in the sent emails. Otherwise `EMAIL_HOST_USER` value will be used.
+
+4. Finally, run `python manage.py migrate drip` to set up the necessary database tables.
+
+## Usage
+
+If you haven't, create a superuser with the [django createsuperuser command](https://docs.djangoproject.com/en/3.0/intro/tutorial02/#creating-an-admin-user). Enter with the admin user, and select `Drips` to manage them. You will be able to:
```

Comment:
Instead of "Enter with.." maybe it would be better something like "Login with..". WDYT?